.github/actionlint.yaml

```diff
@@ -27,6 +27,28 @@ self-hosted-runner:
     - lf.linux.8xlarge.nvidia.gpu
     - lf.linux.16xlarge.nvidia.gpu
     - lf.linux.g5.4xlarge.nvidia.gpu
+    # Organization-wide AWS Linux Runners with new Amazon 2023 AMI
+    - amz2023.linux.large
```

Comment:
It starting to become uncomfortable to double all runners.

Can we come up with a more programatic approach? Like make the lint underrstand that those runners can have the suffix `amz2023.`?

Reply:
Yeah, this is at the limits of acceptability. If we have to double these again, then we'll want to see how to make this more scalable (maybe use codegen to make this file or something).